tests/integration/pidstore/minters/test_minters_arxiv.py

```diff
@@ -59,6 +59,30 @@ def test_minter_arxiv_eprints_empty(base_app, db, create_record_factory):
     assert expected_pids_len == result_pids_len
 
 
+def test_minter_arxiv_eprints_duplicate(base_app, db, create_record_factory):
+    arxiv_value_1 = faker.arxiv()
+    arxiv_value_2 = faker.arxiv()
+    data = {"arxiv_eprints": [{"value": arxiv_value_1}, {"value": arxiv_value_1}]}
```

Comment:
this can't actually happen as the schema has `uniqueItems: true` for all these lists, so the only way there can be two items with the same value is if there are other `properties` in the `object`s to distinguish them.

Reply:
added validation step as well